centreon/src/Core/Dashboard/Infrastructure/API/AddDashboard/AddDashboardPresenter.php

```diff
@@ -63,12 +64,14 @@ public function presentResponse(AddDashboardResponse|ResponseStatusInterface $da
                     [
                         'id' => $data->id,
                         'name' => $data->name,
-                        'description' => $this->emptyStringAsNull($data->description),
-                        'created_by' => $this->userToOptionalArray($data->createdBy),
-                        'updated_by' => $this->userToOptionalArray($data->updatedBy),
+                        'description' => $data->description,
+                        'created_by' => $data->createdBy,
+                        'updated_by' => $data->updatedBy,
```

Comment:
Why this changes ?

Reply:
Instead of using UserResponseDto that is converted to an array, the data are already set as an array. I forgot to delete the UserResponseDto file also, as it is not used anymore.